root_variables.tf

```diff
@@ -2,3 +2,18 @@ variable "tdr_account_number" {
   description = "The AWS account number where the TDR environment is hosted"
   type        = string
 }
+
+variable "dns_zone" {
+  description = "DNS zone name e.g. tdr-management.nationalarchives.gov.uk"
+  default = "tdr-management.nationalarchives.gov.uk"
+}
+
+variable "function" {
```

Comment:
I might just be missing something, but are `var.dns_zone` and `var.function` used anywhere?

Reply:
Thanks, removed both variables as they are not being used